src/test/java/net/fabricmc/mappingio/TestHelper.java

```diff
@@ -20,14 +20,17 @@
 import java.net.URISyntaxException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.concurrent.atomic.AtomicInteger;
 
 import net.neoforged.srgutils.IMappingFile;
 import org.cadixdev.lorenz.io.MappingFormats;
 import org.jetbrains.annotations.Nullable;
+import org.jooq.lambda.Unchecked;
```

Comment:
Do we need a whole new depedency for this? We should only add a new depedency when its really needed. The less depedencies we have the better imo, even if they are test only. It should be trivial to duplicate what you need from this.

Reply:
Copying works for now, but it does become rather cumbersome since each standard-lib functional interface needs an equivalent unchecked one, combined with factory methods, tuples, exception handlers etc I do think it'd make sense to re-add the dependency should we need more of its features in the future.